nvflare/fuel/utils/fobs/decomposer.py

```diff
@@ -58,3 +58,40 @@ def recompose(self, data: Any) -> T:
             The reconstructed object
         """
         pass
+
+
+class DataClassDecomposer(Decomposer):
+    """Generic decomposers for data classes"""
```

Comment:
might be good to explain more or add an example of what is classified as a DataClass

Reply:
I see in README you add it, I think it might be good to also put some of those in docstring

Reply:
Moved the doc from README to code so this feature is treated as internal.